terramate.tm.hcl

```
@@ -3,6 +3,10 @@ terramate {
   # required_version_allow_prereleases = true
   config {
 
+    # Optionally disable safe guards
+    # Learn more: https://terramate.io/docs/cli/orchestration/safeguards
+    # disable_safeguards = ["git-untracked", "git-uncommitted", "outdated-code"]
```

```suggestion
    # disable_safeguards = [
    #  "git-untracked",
    #   "git-uncommitted",
    #  "git-out-of-sync",
    #   "outdated-code",
    # ]
```

`git-out-of-sync` is missing. Also let's use a ordered list here .. please validate this with `terramate fmt` - cheers